shop/myFlywayConfig.conf

```diff
@@ -0,0 +1,5 @@
+flyway.user=sa
```

Comment:
You should define your flyway configuration in the application.properties file. In this way, the migration is done automatically at the application startup.